### DISCUSSION TITLE
Add Autocomplete Mentions

### DISCUSSION THREAD

**Discussion by jsonpreet** [deso-protocol/dips#35] [Medium CIPs]
I tried to make a feature for Autocomplete Mentions in Clouts & Comments, I read more comments related to this feature.

I submitted my PR before the CIP discussion.
https://github.com/bitclout/frontend/pull/268

I am using a package `angular-mentions` for this. Before I tried all manual processes, but I am getting 1 issue, so I thought to use the package.
![Screenshot_3](https://user-images.githubusercontent.com/26606638/125194975-2c755180-e271-11eb-8b83-39a41a502bbf.jpg)

**Comment by lazynina** [comment]
This would be a great change, but at this time, we aren't adding new packages to the frontend until we find a way to protect against software supply chain attacks.

**Reply by jsonpreet** [reply]
Sure i am waiting :)

I will try it without packages, it just have minor issue.

**Reply by jsonpreet** [reply]
You can try live on the testing front end.

https://clouton.io/

**Reply by Octavie** [reply]
@lazynina Is there any news or progress? I was directed here after I created an issue in the FE repo for this.

**Reply by maebeam** [reply]
We are actively working on kickstarting development of LavaMoat's webpack plugin that will allow us to upgrade dependencies safely.

https://github.com/LavaMoat/LavaMoat/issues/253

**Comment by tsheaff** [comment]
@harpreet9629 can you clarify what the issue you ran into doing this manually w/o the package was? Is that WIP attempt code still around in git? I wasn't able to load anything from https://clouton.io/  (I'm getting `Backend returned code 404` error in the console)

It would be nice if we could just fix that one issue, which would uncouple the webpack work from this UX improvement.
